### DISCUSSION TITLE
Enable the Wiki here

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-platform/eclipse.platform.ui#1276] [Ideas]
The old eclipse Wiki is deprecated, but especially in case of E4 there are some useful (but outdated) hints.

I think we should enable the wiki here and just transfer some content (or link to where the content is stored if we want to manage it somewhere else) @vogella WDT can you enable the wiki or open a ticket?

**Comment by akurtakov** [comment]
I've enabled the wiki but please make sure that transfered content is uptodate as just coping outdated things would be disservice to everyone.

**Reply by merks** [reply]
I thought folks such as @mickaelistria have argued that wikis are "poor" and that we should maintain documentation properly...

**Reply by akurtakov** [reply]
That is smth I agree with but I understand how much more work it is to convert to proper docs so we have to give the opportunity here. No one stops anyone from moving wiki content to documentation and make the wiki point to it.

**Reply by akurtakov** [reply]
We see how much effort it takes to get javadocs in proper shape so no technical solution/decision would produce decent docs if there aren't people to actively spend time to make it so.

**Reply by merks** [reply]
That is indeed a fact.    And, in the end, it's not just make it decent but keep it decent...

**Comment by laeubi** [comment]
Yes I know there is always "should we not do this or that" but obviously nothing happens and I'd rather have something not completely optimal than nothing :-)
